src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -460,8 +459,8 @@ protected DynamoModel(StartConfiguration configuration)
             var libraryCore =
                 new ProtoCore.Core(new Options { RootCustomPropertyFilterPathName = string.Empty });
 
-            libraryCore.Executives.Add(Language.kAssociative, new Executive(libraryCore));
-            libraryCore.Executives.Add(Language.kImperative, new ProtoImperative.Executive(libraryCore));
+            libraryCore.Compilers.Add(ProtoCore.Language.kAssociative, new ProtoAssociative.Compiler(libraryCore));
```

Comment:
As these two parameters contain redundant information -- they both convey the information of "associative compiler", I'm thinking probably factory method would be better? For example:
``librarycore.Compiler.Add(CompillerFactory.CreateImperativeCompiler(libraryCore);
librarycore.Compiler.Add(CompillerFactory.CreateAssociativeCompiler(libraryCore);
``